src/Factory/ComponentFactory.php

```diff
@@ -5,6 +5,7 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace PatternBuilder\Factory;
 
 use PatternBuilder\Configuration\Configuration;
```

Comment:
Component (line under this one) isn't used.in a general scope so the line isn't needed. The reference to it in the create method is direct.